### ISSUE TITLE
DESCRIBE KEYSPACE should include Tablets properties 

### ISSUE THREAD

**Issue by tzach** [scylladb/scylladb#18877]
DESCRIBE  KEYSPACE should include the Tablets properties

- DESCRIBE is the easiest way to understand the schema properties
- It is used to restore a backup, so without it a vNode table will be converted to Tablets

**Comment by nyh** [comment]
Please note that the TABLET properties should probably be listed only on DESCRIBE KEYSPACE, not on DESCRIBE TABLE.
You need to set these properties when creating a keyspace - not a table.

**Comment by tzach** [comment]
You are right; I was thinking of moving Tablets properties to Tables, but it's an unnecessary coupling for now.

**Comment by mykaul** [comment]
@piodul , @kostja - who should work on this?

**Comment by kostja** [comment]
We're looking into this, see the relevant discussion in #engineering slack

**Comment by mykaul** [comment]
> We're looking into this, see the relevant discussion in #engineering slack

@kostja - any updates?

**Comment by piodul** [comment]
FWIW, here's the link to the discussion for those who have access to it: https://scylladb-corporate.slack.com/archives/C3AE85S07/p1717576838965149

Summing up, we concluded that there is nothing that needs to be done for DESCRIBE. I also checked manually that the tablet information is inlucded when issuing server-side DESCRIBE KEYSPACE.

There was a concern about tablets and their layout during backup and restore. The plan is to have a cluster-wide snapshot API and then restore data from the saved sstables via load and stream. This is @bhalevy's group domain, and AFAIK @nodep has started work on something adjacent to the cluster-wide snapshot API already.

In any case, there aren't any special additional requirements for DESCRIBE, so I'm closing the issue.
